pkg/logs/config/source.go

```diff
@@ -19,6 +19,10 @@ type LogSource struct {
 	inputs   map[string]bool
 	lock     *sync.Mutex
 	Messages *Messages
+	// sourceType is the type of the source that we are tailing whereas Config.Type is the type of the tailer
+	// that reads log lines for this source. E.g, a sourceType == containerd and Config.Type == file means that
+	// the agent is tailing a file to read logs of a containerd container
+	sourceType string
```

Comment:
nits: sourceType -> \`type\`

Reply:
`type` is a reserved keyword, that's why I put `sourceType`